diff --git a/mps/NLTPSGovernance/tests/nltps.modeltests/models/nltps.modeltests.headless@tests.mps b/mps/NLTPSGovernance/tests/nltps.modeltests/models/nltps.modeltests.headless@tests.mps
index a698ce1..9cb9e6a 100644
--- a/mps/NLTPSGovernance/tests/nltps.modeltests/models/nltps.modeltests.headless@tests.mps
+++ b/mps/NLTPSGovernance/tests/nltps.modeltests/models/nltps.modeltests.headless@tests.mps
@@ -10,6 +10,7 @@
   <imports>
     <import index="pb3e" ref="r:098128cc-3125-4d66-a424-2542a6566f82(nltps.proof.cases)" />
     <import index="icm7" ref="r:b5932721-da09-43aa-899b-0b0afcaf35de(nltps.realization.typesystem)" />
+    <import index="xv4b" ref="r:430deb07-cfa5-41b1-81b1-8b959ab409e8(nltps.governance.typesystem)" />
   </imports>
   <registry>
     <language id="8585453e-6bfb-4d80-98de-b16074f1d86c" name="jetbrains.mps.lang.test">
@@ -91,7 +92,7 @@
           <node concept="1TM$A" id="1bNmcZ3WSdX" role="7EUXB">
             <property role="TrG5h" value="S1_assessed_claim_without_evidence_violates_REA_C_002" />
             <node concept="2PYRI3" id="1bNmcZ3WSdY" role="3lydEf">
-              <ref role="39XzEq" to="icm7:1bNmcZ3TkyF" />
+              <ref role="39XzEq" to="xv4b:1bNmcZ2wLtz" />
             </node>
           </node>
         </node>
diff --git a/mps/NLTPSGovernance/tests/nltps.modeltests/nltps.modeltests.msd b/mps/NLTPSGovernance/tests/nltps.modeltests/nltps.modeltests.msd
index 91133fc..b5623ac 100644
--- a/mps/NLTPSGovernance/tests/nltps.modeltests/nltps.modeltests.msd
+++ b/mps/NLTPSGovernance/tests/nltps.modeltests/nltps.modeltests.msd
@@ -14,6 +14,7 @@
   <dependencies>
     <dependency reexport="false">4745dc6f-8462-405d-b92b-cf0ea82dac23(nltps.proof)</dependency>
     <dependency reexport="false">08070d1a-4999-4bfd-a38c-2b1b3ffd9ef4(nltps.realization)</dependency>
+    <dependency reexport="false">4709dc1d-8658-45c6-b6ee-185bd2ba1b14(nltps.governance)</dependency>
   </dependencies>
   <languageVersions>
     <language slang="l:f3061a53-9226-4cc5-a443-f952ceaf5816:jetbrains.mps.baseLanguage" version="12" />
